db/database_test.go

```diff
@@ -915,6 +916,31 @@ func TestChannelView(t *testing.T) {
 
 }
 
+//////// XATTR specific tests.  These tests current require setting DefaultUseXattrs=true, and must be run against a Couchbase bucket
+
+func CouchbaseTestConcurrentImport(t *testing.T) {
+	db := setupTestDB(t)
+	defer tearDownTestDB(t, db)
+	base.EnableLogKey("Import+")
+
+	// Add doc to the underlying bucket:
+	db.Bucket.Add("directWrite", 0, Body{"value": "hi"})
+
+	// Make sure they aren't visible thru the gateway:
```

Comment:
This comment is confusing .. because the code below seems to be asserting they *are* visible through the sync gateway.